app/views/schools/_form.html.erb

```diff
@@ -1,26 +1,37 @@
 <div class="form-group h3">Create a new School</div>
 <div class='form-group row'>
-  <div class='col-6'>
+  <div class='col'>
       <%= f.label :name, "School Name", class: "label-required" %>
-      <%= f.text_field :name, placeholder: 'UC Berkeley', class: 'form-control',
+      <%= f.text_field :name, placeholder: 'Name of School', class: 'form-control',
```

Comment:
Let's revert these back to what they were. 

In general, a `placeholder` value should be an example of the data, and the label should be the description.